arch/csr/Smcntrpmf/mcyclecfgh.yaml

```diff
@@ -0,0 +1,53 @@
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+$schema: csr_schema.json#
+kind: csr
+name: mcyclecfgh
+long_name: Machine Cycle Counter Configuration High
+address: 0x721
+priv_mode: M
+length: 32
+definedBy: Smcntrpmf
+description: |
+  Upper 32 bits of the 64-bit `mcyclecfg` CSR, used for RV32 systems to access
+  the privilege mode filtering inhibit bits.
+
+fields:
+  MINH:
+    location: 30
+    base: 64
```

Comment:
In RV32, all of the registers are at most 32 bits, so it is impossible to access, say, bit 60 of `mcyclecfg`. Thus, `mcyclecfgh` was created to map the bits in positions higher than 31 to this new register in lower bit positions.

The `base: 64` here would limit the existence of these mapped fields to only RV64, which is obviously not the intent. Does this make sense?

Net: remove the `base: 64` from all of the fields in this register.
`mcyclecfgh` is meant for RV32 tasks to be able to access the